src/Grpc/JsonTranscoding/src/Microsoft.AspNetCore.Grpc.JsonTranscoding/Internal/JsonTranscodingServerCallContext.cs

```diff
@@ -109,6 +110,7 @@ internal async Task ProcessHandlerErrorAsync(Exception ex, string method, bool i
             GrpcServerLog.RpcConnectionError(Logger, rpcException.StatusCode, rpcException.Status.Detail, rpcException.Status.DebugException);
 
             status = rpcException.Status;
+            trailers = rpcException.Trailers;
```

Comment:
Add the trailers from the exception to the `ResponseTrailers` collection (a property on this type) and then use it.

e.g. https://github.com/grpc/grpc-dotnet/blob/1625f8942791c82d700802fc7278c543025f0fd3/src/Grpc.AspNetCore.Server/Internal/HttpContextServerCallContext.cs#L187-L190

Reply:
@JamesNK ResponseTrailersCore currently throws NIE; did you mean to hold this off until it's implemented?